generic/ras.py

```diff
@@ -215,32 +217,35 @@ def test10_ppc64_cpu(self):
         """
         self.log.info("===============Executing ppc64_cpu tool test==========="
                       "====")
-        self.run_cmd("ppc64_cpu --smt")
-        self.run_cmd("ppc64_cpu --smt=8")
-        self.run_cmd("ppc64_cpu --smt")
-        self.run_cmd("ppc64_cpu --smt=4")
-        self.run_cmd("ppc64_cpu --smt")
-        self.run_cmd("ppc64_cpu --smt=off")
-        self.run_cmd("ppc64_cpu --smt")
-        self.run_cmd("ppc64_cpu --smt=on")
-        self.run_cmd("ppc64_cpu --smt")
         self.run_cmd("ppc64_cpu --cores-present")
         self.run_cmd("ppc64_cpu --cores-on")
         self.run_cmd("ppc64_cpu --dscr")
         self.run_cmd("ppc64_cpu --dscr=1")
         self.run_cmd("ppc64_cpu --dscr")
         self.run_cmd("ppc64_cpu --dscr=0")
-        self.run_cmd("ppc64_cpu --smt-snooze-delay")
-        self.run_cmd("ppc64_cpu --smt-snooze-delay=200")
-        self.run_cmd("ppc64_cpu --smt-snooze-delay")
-        self.run_cmd("ppc64_cpu --smt-snooze-delay=100")
-        self.run_cmd("ppc64_cpu --run-mode")
-        self.run_cmd("ppc64_cpu --subcores-per-core")
+        if not IS_POWER_NV:
+            self.run_cmd("ppc64_cpu --run-mode")
+        if not process.system("ppc64_cpu --smt", ignore_status=True):
+            self.run_cmd("ppc64_cpu --smt")
+            self.run_cmd("ppc64_cpu --smt=8")
+            self.run_cmd("ppc64_cpu --smt")
+            self.run_cmd("ppc64_cpu --smt=4")
+            self.run_cmd("ppc64_cpu --smt")
+            self.run_cmd("ppc64_cpu --smt=off")
+            self.run_cmd("ppc64_cpu --smt")
+            self.run_cmd("ppc64_cpu --smt=on")
+            self.run_cmd("ppc64_cpu --smt")
```

Comment:
There is a seperate ppc64_cpu test which does this, and also takes care of supported SMT values. Can you please verify that test, and probably identify the best test to add these to.